docs/scripting/functions/SetPlayerSkillLevel.md

```diff
@@ -46,5 +46,9 @@ The skill parameter is NOT the weapon ID, it is the skill type. Click here for a
 
 ## Related Functions
 
-- SetPlayerArmedWeapon: Set a player's armed weapon.
-- GivePlayerWeapon: Give a player a weapon.
+- [SetPlayerArmedWeapon](../../scripting/functions/SetPlayerArmedWeapon.md): Set a player's armed weapon.
```

Comment:
You're already inside the `scripting` directory, you don't need to go up twice then back in.

Reply:
Thanks for the info, I'll do this for the next pull request since I'm gonna add some hyper links for the letter 'G' since I saw not many of them have hyper links